ebos/equil/initstateequil.hh

```diff
@@ -669,13 +669,20 @@ equil_GOW(const Region& reg, const VSpan& span)
         this->makeOilPressure(ic, reg, span);
     }
 
-    if (this->waterActive()) {
+    if (this->waterActive() && this->oilActive()) {
         // Pcow = Po - Pw => Pw = Po - Pcow
         const auto ic = typename WPress::InitCond {
             reg.zwoc(),
             this->oil(reg.zwoc()) - reg.pcowWoc()
         };
 
+        this->makeWatPressure(ic, reg, span);
+    } else if (this->waterActive() && !this->oilActive()) {
+        // No oil phase set Pw = Pg???
```

Comment:
In this case we **probably** want the gas/water capillary pressure.  Since we define
* Pcgo = Pg - Po
* Pcow = Po - Pw

we can calculate
```
Pcgw = Pg - Pw = Pcgo + Pcow
```
from which we get
```
Pw = Pg - (Pcgo + Pcow)
```

Reply:
Thanks. I will update. In the test case provided by @OPMUSER  pcgo and pcow was both 0.0.